web_widget_timepicker/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Timepicker widget in form view
+==============================
+
+This module defines a timepicker widget, to be used with either char fields
+or (function) fields of type character. Use ``widget='timepicker'`` in your form view
```

Comment:
I agree this widget is relevant. But from your explanation I believe that it would be best for it to work on float fields, and be an alternative to the standard float_time widget.

(Thanks, I'm glad my book helped you)